README.md

```diff
@@ -1,19 +1,238 @@
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be at the time of the
+next major release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes
+should be new RFCs, with a note added to the original RFC. Exactly what counts
+as a "very minor change" is up to the core team to decide.
+
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+While the RFC pull request is up, the team may reach out to the author to 
+discuss the issues in greater detail, and in some cases the topic may be
+discussed at an internal team meeting. In either case a summary from the 
+resulting discussions will be posted back to the RFC pull request.
+
+The core team makes final decisions about RFCs after the benefits and drawbacks
+are well understood. These decisions can be made at any time, but the team
+will regularly issue decisions. When a decision is made, the RFC pull request
+will either be merged or closed. In either case, if the reasoning is not clear
+from the discussion in thread, the team will add a comment describing the
+rationale for the decision.
+
+
+## Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital features that need to be implemented right 
+away. Other accepted RFCs can represent features that can wait until some
+arbitrary developer feels like doing the work. Every accepted RFC has an
+associated issue tracking its implementation in the relevant OBS Project
+repository; thus that associated issue can be assigned a priority via the
+triage process that the team uses for all issues in the relevant repository.
+
+The author of an RFC is not obligated to implement it. Of course, the RFC
+author (like any other developer) is welcome to post an implementation for
+review after the RFC has been accepted.
+
+If you are interested in working on the implementation for an "active" RFC, but
+cannot determine if someone else is already working on it, feel free to ask
+(e.g. by leaving a comment on the associated issue or asking in the
+[official Discord server]).
+
+
+## RFC Postponement
+[RFC Postponement]: #rfc-postponement
+
+Some RFC pull requests are tagged with the "postponed" label when they are
+closed (as part of the rejection process). An RFC closed with "postponed" is
+marked as such because we want neither to think about evaluating the proposal
+nor about implementing the described feature until some time in the future, and
+we believe that we can afford to wait until then to do so. Postponed pull
+requests may be re-opened when the time is right. We don't have any formal
+process for that, you should ask members of the core team.
```

Comment:
The grammar of this sentence seems weird, not sure what it's trying to say. Should we have a formal process for this, or should we just instruct people to ping project members on the closed PR?

Reply:
I don't think having a formal process serves much purpose since it boils down to "when we're ready". I think the intent is that we provide the reason and give a general time-frame in the FCP.